test/helpers/stockLake.ts

```diff
@@ -9,6 +9,6 @@ export function stockLake(...paths: string[]): LakeOfFHIR {
 
 export function restockLake(lake: LakeOfFHIR, ...paths: string[]): void {
   paths.forEach(p => {
-    lake.docs.push(new WildFHIR(fs.readJSONSync(p), p));
+    lake.docs.push(new WildFHIR({ content: fs.readJSONSync(p) }, p));
```

Comment:
I'm not sure how much it really matters since it looks like `restockLake` and `stockLake` are only used in tests, but I think this function would miss adding the `large` boolean to large files. Do you think it's worth adding a similar check for the buffer length there?

Reply:
It wouldn't hurt, and it may actually come in handy for unit tests in the future so I like the idea. I've made this addition in `88c8e31`.